test/system/030-run.bats

```diff
@@ -972,6 +972,9 @@ EOF
     is $output "$randomname" "Should be able to find container by name"
     run_podman rm "/$randomname"
     run_podman 125 create --name "$randomname/" $IMAGE
+
+@test "podman run --net=host --cgroupns=host with read only cgroupfs" {
+    run_podman run --net=host --cgroupns=host --rm -ti $IMAGE grep "^.*/sys/fs/cgroup.*ro.*-.*$" /proc/self/mountinfo
```

Comment:
This will pass on main, because the `ro` will match on the "ro" in "cgROups". A better grep/awk is needed here but I can't tell what that is because, with this patch, I get *two* cgroup lines from grep:
```console
$ bin/podman run --net=host --cgroupns=host --rm $iii grep -E '/sys/fs/cgroup' /proc/self/mountinfo
1529 1527 0:26 / /sys/fs/cgroup rw,nosuid,nodev,noexec,relatime - cgroup2 cgroup2 rw,seclabel
1547 1529 0:26 / /sys/fs/cgroup ro,nosuid,nodev,noexec,relatime - cgroup2 cgroup2 rw,seclabel
                                ^^-- this is the diff
```
(rootless. Root, I get only the last line, the "ro" one).

Also, please remove the `-ti`, it's not needed

Reply:
thanks, I've really messed that up.

The simplest test would be "python -c 'import os; import stat; print(os.statvfs("/sys/fs/cgroup").f_flag & 1)'" but sadly there is no python in the image, and `stat -f` doesn't expose the `f_flag` attribute; so the best I could come with is to retrieve the last `/sys/fs/cgroup` mount and verify it is mounted with `ro`